pagopa/pn-frontend
# Removed IO banner from PG

## Short description
Removed IO banner from PG.

## List of changes proposed in this pull request
- Removed IO banner from notifications and notification details page
- Removed component and tests from repository
- Removed translation properties

## How to test
Open PG, you shouldn't see IO banner anymore in notifications and notification details pages.
